Documentation/ApiOverview/GlobalValues/Typo3ConfVars/SYS.rst

```diff
@@ -0,0 +1,811 @@
+
+.. index::
+   TYPO3_CONF_VARS SYS; loginCopyrightWarrantyURL
+.. _typo3ConfVars_sys_loginCopyrightWarrantyURL:
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['loginCopyrightWarrantyURL']
+===============================================================
+
+.. confval:: loginCopyrightWarrantyURL
+
+   :type: text
+   :Default: ''
+
+   Add the URL where you explain the extend of the warranty you provide.
+   This URL is displayed in the login dialog as the place where people can
+   learn more about the conditions of your warranty. Must be set
+   (more than 10 chars) in addition with the
+   :ref:`loginCopyrightWarrantyProvider<typo3ConfVars_sys_loginCopyrightWarrantyProvider>`
+   message.
+
+.. index::
+   TYPO3_CONF_VARS SYS; textfile_ext
+.. _typo3ConfVars_sys_textfile_ext:
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['textfile_ext']
+==================================================
+
+.. confval:: textfile_ext
+
+   :type: text
+   :Default: 'txt,ts,typoscript,html,htm,css,tmpl,js,sql,xml,csv,xlf,yaml,yml'
+
+   Text file extensions. Those that can be edited. Executable PHP files may not
+   be editable if disallowed!
+
+.. index::
+   TYPO3_CONF_VARS SYS; mediafile_ext
+.. _typo3ConfVars_sys_mediafile_ext:
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['mediafile_ext']
+===================================================
+
+.. confval:: mediafile_ext
+
+   :type: text
+   :Default: 'gif,jpg,jpeg,bmp,png,pdf,svg,ai,mp3,wav,mp4,ogg,flac,opus,webm,youtube,vimeo'
+
+   Commalist of file extensions perceived as media files by TYPO3.
+   Must be written in lower case with no spaces between.
+
+.. index::
+   TYPO3_CONF_VARS SYS; binPath
+.. _typo3ConfVars_sys_binPath:
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['binPath']
+=============================================
+
+.. confval:: binPath
+
+   :type: text
+   :Default: ''
+
+   List of absolute paths where external programs should be searched for.
+   for example :php:`/usr/local/webbin/,/home/xyz/bin/`. (ImageMagick path have to
+   be configured separately)
+
+.. index::
+   TYPO3_CONF_VARS SYS; binSetup
+.. _typo3ConfVars_sys_binSetup:
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['binSetup']
+==============================================
+
+.. confval:: binSetup
+
+   :type: multiline
+   :Default: ''
+
+   List of programs (separated by newline or comma). By default programs
+   will be searched in default paths and the special paths defined by
+   :ref:`binPath<typo3ConfVars_sys_binPath>`. When PHP has :php:`openbasedir`
+   enabled the programs can not be found and have to be configured here.
+
+   Example: :php:`perl=/usr/bin/perl,unzip=/usr/local/bin/unzip`
+
+.. index::
+   TYPO3_CONF_VARS SYS; setMemoryLimit
+.. _typo3ConfVars_sys_setMemoryLimit:
+
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['setMemoryLimit']
+====================================================
+
+.. confval:: setMemoryLimit
+
+   :type: int
+   :Default: 0
+
+   Memory limit in MB: If more than 16, TYPO3 will try to use :php:`ini_set()`
```

Comment:
what is MB?

Reply:
mega byte. But it is usally shortened like that.